src/main.cpp

```diff
@@ -490,7 +493,6 @@ static bool init_common(const Settings &cmd_args, int argc, char *argv[])
 
 	// Initialize sockets
 	sockets_init();
-	atexit(sockets_cleanup);
```

Comment:
why not call uninit_common here ?

Reply:
Using `atexit` you mean? I think it's cleaner when we call it manually.

Reply:
I support nerzhul's request. `atexit` is called reliably on sane program termination, hence should IMO be used for this cleanup.